evalytics/server/models.py

```diff
@@ -6,7 +6,32 @@
 
 
 @dataclass
-class Employee(NodeMixin):
+class Team:
+
+    def __init__(self, name: str,
+                 manager: str = None,
+                 manager_one_level_up: str = None):
+        self.__name = name
+        self.__manager = manager
+        self.__manager_one_level_up = manager_one_level_up
```

Comment:
so this is "who reach first the class, defines the naming convention", just asking

Reply:
hahaha nope, we should end having just one